awx/lib/awx_display_callback/events.py

```diff
@@ -56,9 +54,8 @@ def set(self, key, value):
         filename = '{}-partial.json'.format(event_uuid)
         dropoff_location = os.path.join(self.private_data_dir, 'artifacts', 'job_events', filename)
         write_location = '.'.join([dropoff_location, 'tmp'])
-        partial_data = json.dumps(value)
         with os.fdopen(os.open(write_location, os.O_WRONLY | os.O_CREAT, stat.S_IRUSR | stat.S_IWUSR), 'w') as f:
-            f.write(partial_data)
+            f.write(value)
```

Comment:
so I guess this was already dumped before we got here. That looks like it lines up with the use of the method in this file.